charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -145,13 +217,53 @@ groups:
 
   # Recording rules for the sum of vpa recommendations for all the control-planes
   - record: seed:vpa_status_recommendation_target:sum
-    expr: sum(vpa_status_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"})
+    expr: sum(vpa_status_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"}) by (resource)
   
   # Recording rules for the sum of hvpa applied recommendations for all the control-planes
   - record: seed:hvpa_status_applied_vpa_recommendation_target:sum
-    expr: sum(hvpa_status_applied_vpa_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"})
+    expr: sum(hvpa_status_applied_vpa_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"}) by (resource)
 
   # Recording rules for pod count for all the control-planes
   - record: seed:kube_pod_info:count_cp
     expr: count(kube_pod_info{namespace=~"((shoot-|shoot--)(\\w.+))"})
+  
+  # Recording rules for pod count by status for all the control-planes
+  - record: seed:kube_pod_status_ready:sum_cp_by_condition
+    expr: sum(kube_pod_status_ready{namespace=~"((shoot-|shoot--)(\\w.+))"}) by (condition)
+
+  # Recording rules for pod count by phase for all the control-planes
+  - record: seed:kube_pod_status_phase:sum_cp_by_phase
+    expr: sum(kube_pod_status_phase{namespace=~"((shoot-|shoot--)(\\w.+))"}) by (phase)
 
+  # Recording rules for container count for all the control-planes
+  - record: seed:kube_pod_container_info:count_cp
+    expr: count(kube_pod_container_info{namespace=~"((shoot-|shoot--)(\\w.+))"})
+  
+  # Recording rules for container restart count for all the control-planes
+  - record: seed:kube_pod_container_status_restarts_total:sum_cp
+    expr: sum(kube_pod_container_status_restarts_total{namespace=~"((shoot-|shoot--)(\\w.+))"})
```

Comment:
```suggestion
    expr: sum(kube_pod_container_status_restarts_total{namespace=~"((shoot-|shoot--)(\\w.+))"}) by (namespace)
```

Reply:
@wyb1 Are you sure? This would increase the cardinality of this aggregate metric to the number of shoots running in the seed. Also, the [original usage metrics](https://github.com/gardener/gardener/commit/b646c87ddd8754baa96085d0e937c117d018490f#diff-1751f67b234069990a5af9a44996a6e0R31) didn't have the grouping by namespace either.

Reply:
I remember now. I was thinking `sum_by_cp`. This is the sum of all control planes. My bad.